api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/DataPoint.java

```diff
@@ -18,14 +18,14 @@
 
 import javax.xml.bind.annotation.XmlRootElement;
 
-import com.wordnik.swagger.annotations.ApiClass;
-import com.wordnik.swagger.annotations.ApiProperty;
+import com.wordnik.swagger.annotations.ApiModel;
+import com.wordnik.swagger.annotations.ApiModelProperty;
 
```

Comment:
Did Swagger change the naming between two minor versions? Or do ApiModel and ApiModelProperty have another meaning?